shaluchandwani/Cloth-Store
# Find all clothes

#### What does this PR do?
Add endpoint to get all items
#### Description of Task to be completed?
- Add function to get all items
- Add unit test for the function that helps us to get all items
#### How should this be manually tested?
1. Clone the repo
2. run `npm install`
3. Use postman to test this endpoint this `/api/v1/cloths` and use `3000 port` your url in postman should look like this `localhost:300/api/v1/cloths`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
